### DISCUSSION TITLE
All collections empty after Realm migration

### DISCUSSION THREAD

**Discussion by Speykious** [ppy/osu#16629] [Q&A]
I had a dozen of collections with dozens of maps in them. After migration of the database to Realm, they all became empty.
![image](https://user-images.githubusercontent.com/13885008/151173897-1eb50cf5-608e-4131-8c04-263f31ff7283.png)

Here are all the relevant database files (`.realm` and `.db`)... If any file is missing, let me know. 
[speykious-client-db.zip](https://github.com/ppy/osu/files/7942143/speykious-client-db.zip)

**Comment by peppy** [comment]
Thanks, I'm looking into this one. If you would like to roll back to the previous release in the mean time, you can rename the files you sent to:

`client.db` and `collections.db` (remove the `backup` portion and timestamp) and start an older release for now. Make sure to run the installer for the older release after renaming the files. It will install over the top.

**Reply by Speykious** [reply]
I also found that using the previous `collections.db` on the new release works. I wonder if it should be the case?

**Reply by peppy** [reply]
Yes it actually will, that makes sense. You can also use that.

**Reply by peppy** [reply]
Prospective fix at https://github.com/ppy/osu/pull/16633
